qa/acceptance/Vagrantfile

```diff
@@ -0,0 +1,64 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure(2) do |config|
+
+  config.vm.define "ubuntu-1204" do |ls|
+    ls.vm.box = "elastic/ubuntu-12.04-x86_64"
+    ubuntu_common_config(ls)
+  end
+
+  config.vm.define "ubuntu-1404" do |ls|
+    ls.vm.box = "elastic/ubuntu-14.04-x86_64"
+    ubuntu_common_config(ls)
+  end
+
+  config.vm.define "centos-6" do |ls|
+    ls.vm.box = "elastic/centos-6-x86_64"
+    centos_common_config(ls)
+  end
+
+  config.vm.define "centos-7" do |ls|
+    ls.vm.box = "elastic/centos-7-x86_64"
+    centos_common_config(ls)
+  end
+
+end
+
+def centos_common_config(ls)
+  ls.vm.provider "virtualbox" do |v|
+    v.memory = 4096
+    v.cpus = 4
+  end
+ls.vm.synced_folder "../../build", "/logstash-build"
```

Comment:
I found a small issue when I ran the test, when you start vagrant the `build` folder need to exist before it succesfuly start. Since the `build` is not checked in the repository its possible that the directory wont exist when we try to start the tests.

I suggest for that we try to create it before configuring it.